lib/spark_api/request.rb

```diff
@@ -83,7 +83,7 @@ def request(method, path, body, options)
         request_opts.merge!(options)
         request_path = if middleware && middleware.to_sym == :reso_api
                          dd_version = "Dictionary/#{dictionary_version}/" unless dictionary_version.nil?
-                         "/Reso/#{dd_version}OData#{path}"
+                         "/Version/#{version}/Reso/#{dd_version}OData#{path}"
```

Comment:
Does setting the :reso_api flag set a different default version than the Spark API's "v1"? 

I'm wondering if the default value of this variable needs to be updated on the fly to be compatible with this new RESO API code path:
https://github.com/sparkapi/spark_api/blob/85000035953dbe14149da8801cdc609e44af08ee/lib/spark_api/configuration.rb#L37

Reply:
I agree Dylan. The way this was built was to take into account that parameter under all circumstances. Instead of having a computed default, I opted to ignore this parameter unless it is not the default.